include/libtorrent/socket.hpp

```diff
@@ -93,6 +93,12 @@ namespace libtorrent {
 	using null_buffers = boost::asio::null_buffers;
 #endif
 
+	inline udp::endpoint make_udp(tcp::endpoint const ep)
```

Comment:
do you think it's better to take `const&` instead?

Reply:
I don't really think it matters. the function is making a copy anyway, and it will most likely be inlined and probably disappear entirely